bin/kano-character-cli

```diff
@@ -50,9 +52,17 @@ if configuration is None:
     logger.error('Configuration was not loaded, check configuration file')
     sys.exit(1)
 
+# ERROR RETURN CODES
+SUCCESS = 0
+ERROR_PROCESSING_INPUT_LIST = 2
+COULD_NOT_GENERATE_FINAL_ASSET = 3
+CANT_SELECT_CHARACTER = 4
+CANT_SELECT_ITEMS = 5
+CANT_SELECT_ENV = 6
```

Comment:
I like this, although would it be better putting this in a module so that they can be imported for use by things calling the functions?